<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="c_widget" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>61</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 200, 255);
border-radius: 15px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="m_widget" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>61</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 200);
border-radius: 15px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="y_widget" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>61</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);
border-radius: 15px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="k_widget" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>61</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border-radius: 15px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="lc_widget" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>61</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(120, 255, 255);
border-radius: 15px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="lm_widget" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>61</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 119, 255);
border-radius: 15px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="w_widget" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>61</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="v_widget" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>61</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(128, 128, 128);
border-radius: 15px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="pum_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>339</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 24pt &quot;GOST type A&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>PUMP</string>
    </property>
   </widget>
   <widget class="QLabel" name="sub_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>419</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 24pt &quot;GOST type A&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>SUBT</string>
    </property>
   </widget>
   <widget class="QLabel" name="c_letter">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 16pt &quot;GOST type A&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_letter">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 16pt &quot;GOST type A&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QLabel" name="y_letter">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 16pt &quot;GOST type A&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="k_letter">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 16pt &quot;GOST type A&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QLabel" name="lc_letter">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>270</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 16pt &quot;GOST type A&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Lc</string>
    </property>
   </widget>
   <widget class="QLabel" name="lm_letter">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 16pt &quot;GOST type A&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Lm</string>
    </property>
   </widget>
   <widget class="QLabel" name="w_letter">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 16pt &quot;GOST type A&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QLabel" name="v_letter">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>270</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 16pt &quot;GOST type A&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QWidget" name="c_pum" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="m_pum" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="y_pum" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="k_pum" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="lc_pum" native="true">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>330</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="lm_pum" native="true">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>330</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="w_pum" native="true">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>330</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="v_pum" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>330</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="m_sub" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="y_sub" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="k_sub" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="v_sub" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="c_sub" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="w_sub" native="true">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="lc_sub" native="true">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="lm_sub" native="true">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green;
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="subt_error">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>791</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Arial&quot;;
background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pump_error">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 50pt &quot;Arial&quot;;
background-color: transparent;
color: rgb(255, 0, 4);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>subt_error</zorder>
   <zorder>c_widget</zorder>
   <zorder>m_widget</zorder>
   <zorder>y_widget</zorder>
   <zorder>k_widget</zorder>
   <zorder>lc_widget</zorder>
   <zorder>lm_widget</zorder>
   <zorder>w_widget</zorder>
   <zorder>v_widget</zorder>
   <zorder>pum_label</zorder>
   <zorder>sub_label</zorder>
   <zorder>c_letter</zorder>
   <zorder>m_letter</zorder>
   <zorder>y_letter</zorder>
   <zorder>k_letter</zorder>
   <zorder>lc_letter</zorder>
   <zorder>lm_letter</zorder>
   <zorder>w_letter</zorder>
   <zorder>v_letter</zorder>
   <zorder>c_pum</zorder>
   <zorder>m_pum</zorder>
   <zorder>y_pum</zorder>
   <zorder>k_pum</zorder>
   <zorder>lc_pum</zorder>
   <zorder>lm_pum</zorder>
   <zorder>w_pum</zorder>
   <zorder>v_pum</zorder>
   <zorder>m_sub</zorder>
   <zorder>y_sub</zorder>
   <zorder>k_sub</zorder>
   <zorder>v_sub</zorder>
   <zorder>c_sub</zorder>
   <zorder>w_sub</zorder>
   <zorder>lc_sub</zorder>
   <zorder>lm_sub</zorder>
   <zorder>pump_error</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>

